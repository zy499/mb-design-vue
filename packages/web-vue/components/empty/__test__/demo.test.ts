import demoTest from '../../../scripts/demo-test';

demoTest('empty');
