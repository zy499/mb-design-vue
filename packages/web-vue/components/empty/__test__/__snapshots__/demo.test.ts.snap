// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<empty> demo: render [basic] correctly 1`] = `
"<div class=\\"mb-empty\\">
  <div class=\\"mb-empty-image\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-empty\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
      <path d=\\"M24 5v6m7 1 4-4m-18 4-4-4m28.5 22H28s-1 3-4 3-4-3-4-3H6.5M40 41H8a2 2 0 0 1-2-2v-8.46a2 2 0 0 1 .272-1.007l6.15-10.54A2 2 0 0 1 14.148 18H33.85a2 2 0 0 1 1.728.992l6.149 10.541A2 2 0 0 1 42 30.541V39a2 2 0 0 1-2 2Z\\"></path>
    </svg></div>
  <div class=\\"mb-empty-description\\">暂无数据</div>
</div>"
`;

exports[`<empty> demo: render [custom] correctly 1`] = `
"<div class=\\"mb-empty\\">
  <div class=\\"mb-empty-image\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-exclamation-circle-fill\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
      <path fill-rule=\\"evenodd\\" clip-rule=\\"evenodd\\" d=\\"M24 44c11.046 0 20-8.954 20-20S35.046 4 24 4 4 12.954 4 24s8.954 20 20 20Zm-2-11a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-2a1 1 0 0 0-1-1h-2a1 1 0 0 0-1 1v2Zm4-18a1 1 0 0 0-1-1h-2a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1V15Z\\" fill=\\"currentColor\\" stroke=\\"none\\"></path>
    </svg></div>
  <div class=\\"mb-empty-description\\"> No data, please reload! </div>
</div>"
`;
